src/Andre/SoulsFormats/SoulsFormats/Formats/MSB/MSB3/EventParam.cs

```diff
@@ -885,7 +885,7 @@ public class PatrolInfo : Event
                 /// <summary>
                 /// Unknown; probably some kind of route type.
                 /// </summary>
-                public int UnkT00 { get; set; }
+                public int PatrolType { get; set; }
```

Comment:
maybe separate the msb stuff to a separate PR? Other stuff will need more review

Reply:
I put that stuff into another PR. Would appreciate this being checked over before we get a release out, since this feature has turned out to be QUITE useful.